Jenkinsfile

```diff
@@ -66,7 +84,7 @@ node('rhel8'){
                 def vsix = findFiles(glob: '**.vsix')
                 sh 'vsce publish -p ${TOKEN} --packagePath' + " ${vsix[0].path}"
             }
-            archiveArtifacts artifacts:"**.vsix,**.tgz"
+            archiveArtifacts artifacts:"**.vsix,**.tgz, **/manifest.json"

```

Comment:
I added the manifest.json to be archived. I can create another stage/step if it does not make sense for it to be here.

Reply:
this is fine for now. having it only on release should be enough.